source/concurrent/RecycleBin.ooc

```diff
@@ -10,30 +10,38 @@ use concurrent
 
 RecycleBin: class <T> {
 	_free: Func (T)
-	_list: SynchronizedList<T>
+	_list: VectorList<T>
 	_size: Int
 	count ::= this _list count
 	isFull ::= this count == this _size
 	isEmpty ::= this _list empty
-	init: func (=_size, =_free) { this _list = SynchronizedList<T> new(_size) }
+	_mutex := Mutex new()
```

Comment:
On second thought, this class should be able to use without a mutex, then those properties should be public anyway.